proposals/ml_fit_class_xgboost.json

```diff
@@ -0,0 +1,155 @@
+{
+    "id": "ml_fit_class_xgboost",
+    "summary": "Train an XGBoost classification model",
+    "description": "Executes the fit of an XGBoost classification model based on training data.",
+    "categories": [
+        "machine learning"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "predictors",
+            "description": "The predictors for the XGBoost classification model as a vector data cube. Aggregated to the features (vectors) of the target input variable.",
```

Comment:
> Aggregated to the features (vectors) of the target input variable

Is it necessary to specify this? Isn't it possible to obtain predictors without aggregation?
That sentence reads a bit weird anyway.

Reply:
Not necessary, I'll refactor that.